packages/create-webiny-project/init.js

```diff
@@ -213,10 +213,9 @@ module.exports = async function({ root, appName, templateName, tag, log }) {
                         cwd: root,
                         maxBuffer: "500_000_000"
                     };
-
                     let logStream;
                     if (log) {
-                        logStream = fs.createWriteStream(path.join(root, "logs.txt"), {
+                        logStream = fs.createWriteStream(path.join("./", "cwp-logs.txt"), {
```

Comment:
@EmilK15 what do you think about using `--log` in a way that we need to specify a path to log file? For example:
`--log=./new-project.log` This way the user controls the location of the file.

Reply:
maybe, We can definitely have that functionality present.
Add a bit of description text within the --help function; but if they don't specify a path we could default to the current directory.

Reply:
Yeah, defaults are fine as well.